server/src/models/customers/customer.controller.ts

```diff
@@ -116,4 +123,56 @@ export class CustomerController {
     await this.customerService.deleteCustomer({ id });
     return { id };
   }
+
+  @Auth(Actions.WRITE, [Features.Customer])
```

Comment:
Do we need write access for this? I know customer balance can be sensitive information, this could be something we follow up on in terms of what access this belongs to.

Reply:
Hmm that must be a mistake, updated now